Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/uploadtransformation/TransformationManager.java

```diff
@@ -0,0 +1,38 @@
+package edu.asu.spring.quadriga.service.impl.uploadtransformation;
```

Comment:
package should not be called 'uploadtransformation'. It's more than just about uploading them. Let's call the package transform and add a subpackage 'custom'. Then put this class in that subpackage.